.github/workflows/publish.yml

```diff
@@ -184,10 +157,36 @@ jobs:
 
           helm s3 push webapp-*.tgz charts-webapp
 
+      # generates a mapping between branches/tag to wire-build branches
+      - name: Define target branches in wireapp/wire-builds to bump
+        uses: kanga333/variable-mapper@master
+        id: output_target_branches
+        with:
+          key: '${{github.ref}}'
+          # TODO add production and staging once wire-builds has those branches
+          #
+          # "production": {
+          #  "targets": "[\"TDB\"]"
+          # },
+          # "staging": {
+          #   "targets": "[\"TBD\"]"
+          # },
+          map: |
+            {
+              "dev": {
+                "targets": "[\"dev\"]"
+              },
+              "q1-2024": {
```

Comment:
This will match both a tag that has `q1-2024` or a release branch with that same pattern